index.js

```diff
@@ -6,16 +6,17 @@ import Charts from "./lib/charts";
 if (process.env.SENTRY_DSN) {
   Sentry.init({
     dsn: process.env.SENTRY_DSN,
-    integrations: [Sentry.browserTracingIntegration()],
-    tracesSampleRate: 0,
     allowUrls: [
       /^https:\/\/ecobalyse\.beta\.gouv\.fr/,
       /^https:\/\/staging-ecobalyse\.incubateur\.net/,
+      // Review apps
+      /^https:\/\/ecobalyse-pr.*\.osc-fr1\.scalingo\.io/,
```

Comment:
Why not if there are using the appropriate tag so we can filter production and non production errors on sentry? (this is what this PR does)
It may help debugging?